app/src/main/java/com/kamron/pogoiv/clipboardlogic/ClipboardTokenCollection.java

```diff
@@ -124,11 +127,12 @@
         tokens.add(new HexIVToken()); //hex representation of iv (ex A4B)
         /////////////////////////////////////////////////////////
 
-
-        //Moveset//////////////////////////////////////
-        tokens.add(new MovesetInitialsToken(1));
-        tokens.add(new MovesetInitialsToken(2));
-
+        //enable/disable visibility based on setting
+        if (GoIVSettings.getInstance(context).isMovesetEnabled()) {
```

Comment:
This method and the list it returns shouldn't be changed by the user's preferences because this same list is used to read/write tokens configuration into sharedpreferences.
This list should never change its contents (aside from new token additions when a new app version is released) or the token configuration logic would fail.